### PR TITLE
Update nature.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/79336/style-error-nature-and-possibly-subjournals
Just submitting this. Your decision to take it or not :)

### CODE REVIEW

**Comment by adam3smith** [comment]
(I do want to take this, but unsure of the format. The semicolon seems odd, but is in the published articles. The Lake Forrest version also seems a bit off to me. I'll want to explore some more samples I think.)

**Comment by adam3smith** [comment]
Looked around a bit e.g. at https://www.nature.com/articles/s41586-019-1619-z reference 42 and I think the format suggested by the LFC guide, i.e. title. URL (year) is the way to go. I'd be inclined to just change website to the same format. I didn't see an access date in any of the articles I looked at (LFC suggests using it for frequently changing sites)

File: nature.csl [hunk 1]
```diff
@@ -111,9 +120,9 @@
             <text term="in"/>
           </if>
         </choose>
-        <text macro="container-title"/>
+        <text macro="container-title" suffix="."/>
```
Comment (adam3smith) [inline nature.csl]: I'm not seeing that for journals. Where is the period coming from?

Comment (POBrien333) [reply nature.csl]: I put this in for something, but can't remember now.
The way of doing this is wrong anyway. Not every container title has dots at the end (that's what I was thinking).

**Pull request merged**